### DISCUSSION TITLE
Welcome to sage Discussions!

### DISCUSSION THREAD

**Discussion by lazear** [lazear/sage#108] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FriedLabJHU** [comment]
Hello, proteomics people!

I am Edgar Manriquez-Sandoval, a 4th yr PhD student at JHU in Stephen Fried's lab.
I use mass spectrometry and bottom-up proteomics to study protein folding processes. Currently, I am trying to understand the evolutionary relationship between protein sequences and protein refolding.

This community has been my favorite place to discuss ideas and talk shop about science, tools, and omics, so I am always happy to chat! Besides my research, I developed proteomic tools for annotating protein domains and, recently, for analyzing limited-proteolysis data that I hope people find useful.

I believe that developing novel tools (like sage!) is the only way we will push past our current boundaries in proteomics.

**Comment by theGreatHerrLebert** [comment]
Hey everyone,

I am David Teschner, a 4th-year PhD student in the field of scientific computing and bioinformatics at JGU Mainz in Germany, working in the group of Andreas Hildebrandt. My focus has been on algorithm and software development for mass spectrometry devices that utilize ion mobility as an additional dimension of separation. This is mostly in developing new processing techniques for DIA methods like DIA-PASEF or MIDIA.

While I am passionate about science, I believe with some exceptions the way scientific software is managed post-publication is often why many lab professionals still prefer proprietary software over open solutions. SAGE is one of these exceptions. I had the pleasure of learning a great deal by going through its code and understanding the identification problem with the resources provided. Looking forward to developing with you guys!
